drivers/boards/BLACKPILL_STM32_F411/bootloader_defs.h

```diff
@@ -0,0 +1,7 @@
+/* Address for jumping to bootloader on STM32 chips. */
+/* It is chip dependent, the correct number can be looked up here:
+ * http://www.st.com/web/en/resource/technical/document/application_note/CD00167594.pdf
+ * This also requires a patch to chibios:
+ *  <tmk_dir>/tmk_core/tool/chibios/ch-bootloader-jump.patch
+ */
+#define STM32_BOOTLOADER_ADDRESS 0x1FFF0000
```

Comment:
I think I'd prefer to remove this file entirely and use it from mcu_selection.mk -- we already have a variable set up for this.